corehq/apps/app_manager/helpers/validators.py

```diff
@@ -552,6 +563,9 @@ def validate_details_for_build(self):
                             'module': self.get_module_info(),
                             'reason': _('A case property must be assigned to the "{}" tile field.'.format(field))
                         })
+            self._validate_fields_with_format('address', 'Address', detail.columns, errors)
```

Comment:
I don't think there are python constants for those. Let me know if I am mistaken.